doc/classes/CubemapArray.xml

```diff
@@ -1,8 +1,19 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="CubemapArray" inherits="TextureLayered" version="4.0">
 	<brief_description>
+		A [TextureLayered] of type [code]enum TextureLayeredType.TEXTURE_LAYERED_CUBEMAP_ARRAY[/code] which uses 6, or a multiple of 6, layers.
 	</brief_description>
 	<description>
+		[TextureLayered]'s and [Cubemap]'s class references explains how a [CubemapArray] operates and how it has to be handled.
+		It is based on the class [Texture] (not [Texture2D]) and can only used as such. Its internal resource type, as well as recognized file extension, is [code]cubearr[/code].
+		The [CubemapArray] is an array of [Image]s, not cubemap levels (ie. [Cubemap]s). In Godot, technically, the [CubemapArray] is the same as the [Cubemap]. The only difference is that the [b]cubemap array has more layers[/b].
```

Comment:
I would be careful in this statement. It _is_ an array of cubemap levels. It just so happens that a cubemap is created with an array of 6 images. But internally it is a single resource. 

You are confusing how the texture is created vs. what it actually _is_.

Reply:
You're right, it is confusing. I hope the most recent commit makes it clearer